lib/metadata/wrapper.js

```diff
@@ -2,7 +2,6 @@ const MetadataWrapper = require('arsenal').storage.metadata.MetadataWrapper;
 const { config } = require('../Config');
 const logger = require('../utilities/logger');
 const constants = require('../../constants');
-const bucketclient = require('bucketclient');
```

Comment:
From my pov, this is still needed :/ the MetadataWrapper is expecting this as a param

Reply:
question: why ?